circle.yml

```diff
@@ -0,0 +1,18 @@
+machine:
```

Comment:
Oh this was an attempt to get the 0.13.x branch to build on Circle, AFAIK it was working on Travis before.